components/Members/MemberOverview.tsx

```diff
@@ -160,6 +162,11 @@ const MemberOverview = ({ member }: { member: Member }) => {
   const paginateVotes = (page) => {
     return ownVoteRecords.slice(page * perPage, (page + 1) * perPage)
   }
+
+  const abbreviatePublicKey = (publicKey: string) => {
```

Comment:
nit: don't we have a function for this already? Ideally all address should be displayed using a component with a link to explorer

Reply:
removed function